pkg/api/v1/helper/helpers.go

```diff
@@ -433,6 +433,26 @@ func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
 	return ""
 }
 
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "".
```

Comment:
Why not return Filesystem mode by default?

Reply:
I guess I could do that, doesn't look like any of our binding logic cases from the matrix would be impacted by defaulting this.  So this would basically turn our matrix into this

```
PV volumeMode	PVC volumeMode	Result
Block		Block		BIND
Filesystem	Filesystem	BIND

Filesystem	Block		NO BIND 
Block		Filesystem	NO BIND 
```

agree?

Reply:
yup